types/vote.go

```diff
@@ -85,16 +85,17 @@ func (vote *Vote) String() string {
 		cmn.PanicSanity("Unknown vote type")
 	}
 
-	return fmt.Sprintf("Vote{%v:%X %v/%02d/%v(%v) %X %X @ %s}",
-		vote.ValidatorIndex,
-		cmn.Fingerprint(vote.ValidatorAddress),
-		vote.Height,
-		vote.Round,
+	return fmt.Sprintf("Vote{%v(%v) %v/%02d @ %s %X %v:%X   %X}",
```

Comment:
Oh I don't think we should change the order in the print statement. It's super convenient for humans to have the validator info at the front